cli/args/config_file.rs

```diff
@@ -66,7 +66,7 @@ pub struct IgnoredCompilerOptions {
 }
 
 impl fmt::Display for IgnoredCompilerOptions {
-  fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+  fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
```

Comment:
Kind of seems like a step back. I like hidden lifetimes.

Reply:
OK, if that's a project-wide decision, what about making it explicit? e.g.

```diff
diff --git .cargo/config.toml .cargo/config.toml
index cac99a377..f52833b14 100644
--- .cargo/config.toml
+++ .cargo/config.toml
@@ -23,4 +23,8 @@ rustflags = [
   "clippy::missing_safety_doc",
   "-D",
   "clippy::undocumented_unsafe_blocks",
+  "-D",
+  "rust-2018-idioms",
+  "-A",
+  "elided-lifetimes-in-paths",
 ]
```

Or maybe even just `-A elided-lifetimes-in-paths` if `rust-2018-idioms` isn't deemed useful.